test/models/host_test.rb

```diff
@@ -3577,6 +3577,22 @@ def to_managed!
     end
   end
 
+  describe 'HostParams module' do
+    test 'shows global params' do
+      host = FactoryGirl.build(:host)
+      parameter = FactoryGirl.create(:common_parameter, :name => 'test_param1', :value => 'test_value1')
+
+      assert_equal 'test_value1', host.host_params['test_param1']
+    end
+
+    test 'renders global params if template specified' do
+      host = FactoryGirl.build(:host)
+      parameter = FactoryGirl.create(:common_parameter, :name => 'test_param1', :value => '<%= "test_value1" %>')
```

Comment:
nitpick: not that matters much, but i think it would be more interesting to use some more computed value.